src/algebra/algebra/subalgebra.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau, Yury Kudryashov
 -/
 import algebra.algebra.basic
+import algebra.algebra.operations
```

Comment:
I guess that this transitively imports `algebra.algebra.basic`, so you can remove the import above this one.

Reply:
I don't know what the lean mantra is here, but in C++ the approach is to pretend that transitive imports don't exist, and always import things you directly use, so that if transitive imports change nothing gets broken.

Reply:
Oh ok, I did not know that. Thank you!